### PR TITLE
Taxi example files

### PR DESCRIPTION
**Pull request by joshuawonghpe**
Contains notebook files for both Kubernetes and Epic MLOps workflows.

### CODE REVIEW

File: MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  endLong = input("Please indicate the longitude of the dropoff point (between -73.750 and -75.250): ")
+  miles = input("Please indicate the approx trip distance in miles: ")
+  weekday = input("Please enter 1 if the trip will occur on a weekday, 0 otherwise: ")
+  hour = input("Please indicate what hour of day the trip occurred (range: 0 to 23): ")
+
+  url = accessUrl + modelName + "/" + modelVersion + "/predict?X-Auth-Token=" + authToken
+  url = accessUrl + modelName + "/" + modelVersion + "/predict?X-Auth-Token=" + authToken
+
+  payload = "{\r\n    \"use_scoring\": true,\r\n    \"scoring_args\": {\r\n        \"work\": " + work + "," \
+            "\r\n        \"startstationlatitude\": " + startLat + "," \
```
Comment (kmathur2) [inline MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py:28]: any reason to stringify this?

File: MLOps/examples/NYCTaxi/Taxi Datasets/lookup-ipyheader.csv [hunk 2]
```diff
@@ -0,0 +1,264 @@
+pqlookup.long,pqlookup.lat,pqlookup.borough,pqlookup.location_i,pqlookup.objectid,pqlookup.shape_area,pqlookup.shape_leng,pqlookup.zone
```
Comment (kmathur2) [inline MLOps/examples/NYCTaxi/Taxi Datasets/lookup-ipyheader.csv:1]: In your readme, please describe the data also

File: MLOps/examples/NYCTaxi/XGBoost Demo/ExampleAPICallInfoForPostman.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+POST url: http://mip-ap23-vm02.mip.storage.hpecorp.net:10039/Taxi_Duration/1/predict
+
+
+API body:
+{
```
Comment (kmathur2) [inline MLOps/examples/NYCTaxi/XGBoost Demo/ExampleAPICallInfoForPostman.txt:5]: please add comments to describe f0-f13

File: MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  startLat = input("Please indicate the latitude of the pickup point (between 40.550 and 40.925): ")
+  startLong = input("Please indicate the longitude of the pickup point (between -73.750 and -75.250): ")
+  endLat = input("Please indicate the latitude of the dropoff point (between 40.550 and 40.925): ")
+  endLong = input("Please indicate the longitude of the dropoff point (between -73.750 and -75.250): ")
+  miles = input("Please indicate the approx trip distance in miles: ")
+  weekday = input("Please enter 1 if the trip will occur on a weekday, 0 otherwise: ")
+  hour = input("Please indicate what hour of day the trip occurred (range: 0 to 23): ")
+
+  url = accessUrl + modelName + "/" + modelVersion + "/predict?X-Auth-Token=" + authToken
+  url = accessUrl + modelName + "/" + modelVersion + "/predict?X-Auth-Token=" + authToken
```
Comment (kmathur2) [inline MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py:25]: nit: duplicate url

File: MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py (hunk 1 above)
Comment (joshuawonghpe) [reply MLOps/examples/NYCTaxi/TensorFlow/query_api_script_tf.py]: Good point, I have changed it so it is not a string anymore.

File: MLOps/examples/NYCTaxi/Taxi Datasets/lookup-ipyheader.csv (hunk 2 above)
Comment (joshuawonghpe) [reply MLOps/examples/NYCTaxi/Taxi Datasets/lookup-ipyheader.csv:1]: I have just added a description of the data.

**Pull request merged**